mpp/mpp_efp.F90

```diff
@@ -697,7 +697,7 @@ subroutine mpp_efp_list_sum_across_PEs(EFPs, nval, errors)
     do n=1,NUMINT ; EFPs(i)%v(n) = ints(n,i) ; enddo
     if (present(errors)) errors(i) = overflow_error
     if (overflow_error) then
-      write (mesg,'("mpp_efp_list_sum_across_PEs error at ",i6," val was ",ES12.6, ", prec_error = ",ES12.6)') &
+      write (mesg,'("mpp_efp_list_sum_across_PEs error at ",i6," val was ",ES13.6, ", prec_error = ",ES13.6)') &
```

Comment:
Is this a change to an error message?

Reply:
Yeah, format code was throwing a warning, this just changes how the number is printed